### PR TITLE
Feature/iconfont toolbar

### PR DESCRIPTION
**Pull request by FinalAngel**
image

### CODE REVIEW

**Comment by vxsx** [comment]
whoops sorry, mobile interface is too sensitive

**Comment by FinalAngel** [comment]
no problem, take it easy

File: cms/static/cms/sass/components/_toolbar.scss [hunk 1]
```diff
@@ -430,9 +443,15 @@
         margin-top: 9px;
         transform: rotate(90deg);
     }
-    .cms-toolbar-loader a {
-        background: #fcfcfc url("../img/loader.gif") no-repeat center center !important;
-        background-size: 20px 20px !important;
+    .cms-toolbar-loader a span {
+        @extend .cms-icon-loader;
```
Comment (vxsx) [inline cms/static/cms/sass/components/_toolbar.scss]: here you extend `cms-icon-loader`, but not `cms-icon` like in logo for example
which approach is correct?

Comment (FinalAngel) [inline cms/static/cms/sass/components/_toolbar.scss]: das correct, per default it has already ``cms-icon cms-icon-arrow``

File: cms/templates/cms/toolbar/items/item_ajax.html [hunk 2]
```diff
@@ -1,3 +1,5 @@
 <li class="{% if active %}cms-toolbar-item-navigation-active{% endif %}{% if disabled %} cms-toolbar-item-navigation-disabled{% endif %} {{ extra_classes|join:' ' }}">
-	<a href="{{ action }}" data-rel="ajax" data-post="{{ data }}"{% if question %} data-text="{{ question }}"{% endif %}{% if on_success %} data-on-success="{{ on_success }}"{% endif %}><span>{{ name }}</span></a>
+	<a href="{{ action }}" data-rel="ajax" data-post="{{ data }}"{% if question %} data-text="{{ question }}"{% endif %}{% if on_success %} data-on-success="{{ on_success }}"{% endif %}>
+	    <span>{{ name }}<span class="cms-icon cms-icon-arrow"></span></span>
```
Comment (vxsx) [inline cms/templates/cms/toolbar/items/item_ajax.html]: here you are putting an icon inside a span, but in `cms/templates/cms/toolbar/items/item_link.html` you put it next to the span, I would prefer latter approach everywhere

File: cms/static/cms/sass/components/_toolbar.scss (hunk 1 above)
Comment (vxsx) [inline cms/static/cms/sass/components/_toolbar.scss]: ah, I see

File: cms/templates/cms/toolbar/items/item_ajax.html (hunk 2 above)
Comment (FinalAngel) [inline cms/templates/cms/toolbar/items/item_ajax.html]: fixed everywhere

File: cms/templates/cms/toolbar/items/menu.html [hunk 3]
```diff
@@ -1,7 +1,5 @@
-<li{% if sub_level %} class="cms-toolbar-item-navigation-children {% if active %}cms-toolbar-item-navigation-active{% endif %}"{% endif %}><a href="{{ url }}"><span>{{ title }}</span></a>
-	{% if items %}
```
Comment (vxsx) [inline cms/templates/cms/toolbar/items/menu.html]: is it ok to remove this?

Comment (FinalAngel) [inline cms/templates/cms/toolbar/items/menu.html]: it's not removed O_o

**Comment by vxsx** [comment]
LGTM, great job!

**Pull request merged**